### PR TITLE
test: Modify a regex-assembly file to test the linter

### PR DESCRIPTION
**Pull request by RedXanadu**
IGNORE.

Testing that linting of regex-assembly files is working correctly. IGNORE.

### CODE REVIEW

**Comment by RedXanadu** [comment]
This is a problem:

I can change a .ra file without re-compiling the regex in the rule / rule file;
I can change a regex in a rule without amending the .ra file.

Neither situation is detected by the GitHub Action checks / linter. This is a problem.

File: rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf [hunk 1]
```diff
@@ -168,7 +168,7 @@ SecRule REQUEST_HEADERS:Content-Length "!@rx ^\d+$" \
 # https://www.rfc-editor.org/rfc/rfc9110.html#section-9.3.1
 # https://www.rfc-editor.org/rfc/rfc9110.html#section-9.3.2
 #
-SecRule REQUEST_METHOD "@rx ^(?:GET|HEAD)$" \
+SecRule REQUEST_METHOD "@rx ^(?:GET|HEAD|ASDF)$" \
```
Comment (fzipi) [inline rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf:171]:
```suggestion
SecRule REQUEST_METHOD "@rx ^(?:GET|HEAD|ASDF|SOMETHINGTORUNTHEPIPELINEAGAIN)$" \
```

**Comment by RedXanadu** [comment]
Double/triple-checking.

Confirmed that modifying a .ra file without updating the associated rule regex is caught:

```
Error: All rules need to be up to date. Please run `crs-toolchain regex update --all`
Error: Process completed with exit code 1.
```

**Comment by RedXanadu** [comment]
Confirmed that directly modifying a compiled regex in a rule is also caught as expected:

```
Error: All rules need to be up to date. Please run `crs-toolchain regex update --all`
Error: Process completed with exit code 1.
```

**Comment by RedXanadu** [comment]
This is all now working as expected. Closing.

**Pull request closed without merging**